src/linker/Linker.Steps/TrimUsingVisibleMembers.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Runtime.CompilerServices;
+using Mono.Cecil;
+
+namespace Mono.Linker.Steps
+{
+	public class TrimUsingVisibleMembers : BaseStep
```

Comment:
Same comment as for the entry point one - maybe call this `RootVisibleMembersStep`?

Reply:
Changed the approach in follow up PR